src/main/java/uk/nhs/tis/trainee/usermanagement/api/UserAccountResource.java

```diff
@@ -129,4 +136,66 @@ ResponseEntity<Void> deleteCognitoAccount(@PathVariable String username) {
     log.info("Deleted Cognito account for user '{}'.", username);
     return ResponseEntity.noContent().build();
   }
+
+  /**
+   * Add the given user into Cognito user group.
+   *
+   * @param usergroup The name of the Cognito user group.
+   * @param username The username of the user.
+   * @return 204 No Content, if successful.
+   */
+  @PostMapping("/add-usergroup")
+  ResponseEntity<Void> addToUserGroup(
+      @RequestParam(value = "usergroup", required = true) String usergroup,
+      @RequestParam(value = "username", required = true) String username
```

Comment:
As below, don't let the caller set the user group.

Reply:
Updated.